src/modules/core/components/TokenEditDialog/TokenEditDialog.tsx

```diff
@@ -46,50 +47,74 @@ const MSG = defineMessages({
 });
 
 interface Props {
-  addTokenFn: (address: Address) => Promise<any>;
+  updateTokens: (addresses: Address[]) => Promise<any>;
   cancel: () => void;
   close: () => void;
+  // Colony tokens
   tokens: AnyToken[];
+  // Token list from json file. Not supported on local env
+  tokensList?: AnyToken[];
+  // Colony native token addresss
+  nativeTokenAddress?: Address;
 }
 
 interface FormValues {
   tokenAddress: Address;
   description?: string;
+  tokenAddresses: Address[];
 }
 
 const validationSchema = yup.object({
-  tokenAddress: yup
-    .string()
-    .address()
-    // @todo validate against entering a duplicate address
-    .required(),
-  tokenSymbol: yup.string().max(6),
+  tokenAddress: yup.string().address(),
 });
 
-const TokenEditDialog = ({ addTokenFn, tokens = [], cancel, close }: Props) => {
-  const { formatMessage } = useIntl();
+const TokenEditDialog = ({
```

Comment:
Can we change the "Add Token" input to use the `--family-techie` font. I think it will better align with the way we've been displaying token addresses:

![Screenshot from 2020-12-18 15-44-00](https://user-images.githubusercontent.com/1193222/102621256-fd1dbc80-4147-11eb-9a88-74753edb301f.png)

To look more like this:

![Screenshot from 2020-12-18 15-47-28](https://user-images.githubusercontent.com/1193222/102621526-61d91700-4148-11eb-8d59-cec12d726e5d.png)

Maybe ask Karol if he's ok with this?